cfme/fixtures/multi_region.py

```diff
@@ -0,0 +1,75 @@
+import attr
+import pytest
+
+
+@attr.s
+class ApplianceCluster(object):
+    global_appliance = attr.ib(default=None)
+    remote_appliances = attr.ib(default=[])
+
+
+@pytest.fixture(scope='module')
+def multi_region_cluster(temp_appliances_unconfig_modscope_rhevm):
+    # todo: make it work for all appliance types
+    appliances = temp_appliances_unconfig_modscope_rhevm
+    cluster = ApplianceCluster()
+
+    cluster.global_appliance = appliances[0]
+    cluster.remote_appliances = appliances[1:]
```

Comment:
Q: Why `remote_appliances`? In what instance is there more than 1?

Reply:
there should be such examples in future. usually such configuration have a bunch of remote regions' appliances and one global region appliance.